Tests/PersistenceBaseTest.m

```diff
@@ -11,32 +11,63 @@
 @interface PersistenceBaseTest ()
 
 @property (nonatomic) CDAPersistenceManager* persistenceManager;
-@property (nonatomic) NSDate* lastSyncTimestamp;
 
 @end
 
 #pragma mark -
 
 @implementation PersistenceBaseTest
 
+-(void)setUp {
+    [super setUp];
+
+    self.lastSyncTimestamp = nil;
+
+    [self buildPersistenceManagerWithDefaultClient:NO];
+    // MUST BE CALLED SO THAT THE STUBS ARE ACTUALLY RETURNED.
+    [self setUpCCLRequestReplayForNSURLSession];
+
+    [self deleteStore];
+}
+
+-(void)tearDown {
+    [super tearDown];
+
+    self.persistenceManager = nil;
+}
+
+- (void)deleteStore {
+    [NSException raise:@"Delete not implemented" format:@"Must implement deleteStore method"];
+}
+
 -(void)assertNumberOfAssets:(NSUInteger)numberOfAssets numberOfEntries:(NSUInteger)numberOfEntries {
     XCTAssertEqual(numberOfAssets, [self.persistenceManager fetchAssetsFromDataStore].count, @"");
     XCTAssertEqual(numberOfEntries, [self.persistenceManager fetchEntriesFromDataStore].count, @"");
 
+
+    // FIXME: make this is a helper method that returns a BOOl and then check for equality elsewhere.
```

Comment:
Good point. Maybe we should take this comment out and make it more obvious, aka in a Github issue, maybe?